Queries/SecondQuery.sql

```diff
@@ -0,0 +1,30 @@
+WITH MoreThanOneUnReturnedBooks
+AS (
+	SELECT 
+	   COUNT(*) AS numOfUnReturned,
+	   b.BorrowerID
+	FROM 
+	   Borrowers AS b
+	JOIN 
+	   Loans AS l ON b.BorrowerID = l.BorrowerID
+	WHERE 
+	   l.DateReturned IS NULL
+	GROUP BY 
+	   b.BorrowerID
+	HAVING
+	   COUNT(*) >= 2
+	)
+
+SELECT
+   b.BorrowerID, 
+   CONCAT (b.FirstName, ' ', b.LastName) AS [Borrower Name], 
+   numOfUnReturned AS [Number of un returned books]
+FROM
+   MoreThanOneUnReturnedBooks AS unreturned
+JOIN 
+   Borrowers AS b ON b.BorrowerID = unreturned.BorrowerID
+
+
+
+
+ 
```

Comment:
Since the first and last names are in borrower table, we can do like:

```suggestion 
WITH MoreThanOneUnReturnedBooks AS (
    SELECT 
        COUNT(*) AS numOfUnReturned,
        b.BorrowerID,
       CONCAT (b.FirstName, ' ', b.LastName) AS [Borrower Name]
    FROM 
        Borrowers AS b
    JOIN 
        Loans AS l ON b.BorrowerID = l.BorrowerID
    WHERE 
        l.DateReturned IS NULL
    GROUP BY 
        b.BorrowerID, b.FirstName, b.LastName
    HAVING
        COUNT(*) >= 2
)
```